lib/entice/logic/movement.ex

```diff
@@ -28,6 +29,9 @@ defmodule Entice.Logic.Movement do
         |> Map.put(Position, new_pos)
         |> Map.put(Movement, new_movement)
       end)
+    {:ok, %MapInstance{map: map, players: _}} = Entity.fetch_attribute(entity, MapInstance)    
+    #Can't figure out how to get eid from pid
+    Coordination.notify_all(map, {:movement_agent_updated, new_pos, entity}) #TODO: Change to use eid instead of pid for self seeking guard in seek l30
```

Comment:
Do we need this? Or can you maybe match on state changes later on like in https://github.com/entice/web/blob/develop/web/channels/entity_channel.ex#L74-L78